vehicles/hovercraft.lua

```diff
@@ -0,0 +1,20 @@
+local definition = ...
+local types = {
+    "blue", "green", "red", "yellow"
+}
+
+for _, color in pairs(types) do
+    if minetest.settings:get_bool("vehicle_mash.enable_hover_" .. color) == false then return end
+
+    local name = "hover_"..color
+    local capitalization = color:gsub("^%l", string.upper)
+    local style = capitalization:gsub("_", " ")
+
+    definition.description = vehicle_mash.S("@1 Hovercraft", style)
```

Comment:
`local definition = table.copy(definition);`. Otherwise you end up modifying `def` table in https://github.com/minetest-mods/vehicle_mash/blob/master/framework.lua#L89 .